main.py

```diff
@@ -163,9 +165,29 @@ def handle_errors(self, event):
         The event is sent only if the error is related to a request made
         by flow_manager.
         """
-        xid = event.content["message"].header.xid.value
-        error_type = event.content["message"].error_type
-        error_code = event.content["message"].code
+        message = event.content["message"]
+
+        connection = event.source
+        switch = connection.switch
+
+        xid = message.header.xid.value
+        error_type = message.error_type
+        error_code = message.code
+        error_data = message.data.pack()
+
+        # Get the packet responsible for the error
+        error_packet = connection.protocol.unpack(error_data)
+
+        if message.code == BadActionCode.OFPBAC_BAD_OUT_PORT:
+            for action in error_packet.actions:
+                try:
+                    iface = switch.get_interface_by_port_no(action.port.value)
+                except AttributeError:
+                    iface = switch.get_interface_by_port_no(action.port)
+
+                # Check interface to drop packets forwarded to it
```

Comment:
"Check" gets ambiguous here, meaning "verify interface". "Set" or "mark" should be more clear.

```suggestion
                # Set interface to drop packets forwarded to it
```

Reply:
Ok